grid2op/Runner/Runner.py

```diff
@@ -479,19 +488,28 @@ def _new_env(self, chronics_handler, backend, parameters):
                             chronics_handler=chronics_handler,
                             backend=backend,
                             parameters=parameters,
+                            name=self.name_env,
                             names_chronics_to_backend=self.names_chronics_to_backend,
                             actionClass=self.actionClass,
                             observationClass=self.observationClass,
                             rewardClass=self.rewardClass,
                             legalActClass=self.legalActClass,
                             voltagecontrolerClass=self.voltageControlerClass,
                             other_rewards=self._other_rewards,
+
                             opponent_action_class=self.opponent_action_class,
                             opponent_class=self.opponent_class,
                             opponent_init_budget=self.opponent_init_budget,
                             opponent_budget_per_ts=self.opponent_budget_per_ts,
                             opponent_budget_class=self.opponent_budget_class,
-                            name=self.name_env)
+                            opponent_attack_duration=self.opponent_attack_duration,
+                            opponent_attack_cooldown=self.opponent_attack_cooldown,
+                            kwargs_opponent=self.opponent_kwargs,
+
```

Comment:
indent

Reply:
I don't see indend issue on pycharm for windows :-/